lib/reducers/CoreValueReducer.js

```diff
@@ -17,14 +17,11 @@ const characterReducer = (state = coreValueData, action) => {
     const currentValue = currentValues[action.coreValue];
     const updatedStudent = Object.assign({}, currentValues, {[coreValueToUpdate]: currentValue + 1});
     return Object.assign({}, state, {[action.studentId]: updatedStudent} );
-  // case 'SEND_SUMMARIES':
-  //
-  //   const reduced = action.student.reduce((acc, points) => {
-  //     points + acc;
-  //     return acc;
-  //   }, 0);
-  //
-  //   return reduced;
+  case 'SEND_SUMMARIES':
+    const pointValues = Object.values(state);
+    return pointValues.reduce((acc, points) => {
+      return points + acc;
+    }, 0);
```

Comment:
This is cool! I haven't seen a reduce in a reducer before. A reducerer, if you will.

Reply:
<3